.github/workflows/sync-supported-chains.yml

```diff
@@ -7,37 +7,73 @@ on:
     types:
       - completed
   workflow_dispatch:
+    inputs:
+      nethermind_image:
+        description: "Docker image to be used by action"
+        default: "nethermindeth/nethermind:master"
+        required: true
 
 env:
   DOTNET_SYSTEM_CONSOLE_ALLOW_ANSI_COLOR_REDIRECTION: "1"
   TERM: xterm
 
 jobs:
-  testnet-sync:
+  setup-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: nethermind
+      - name: Set Matrix
+        id: set-matrix
+        run: echo "matrix=$(jq -c . nethermind/scripts/workflow_config/sync_testnets_matrix.json)" >> $GITHUB_OUTPUT
+
+  create_a_runner:
+    needs: [setup-matrix]
+    strategy:
+      fail-fast: false
+      matrix:
+        client-network: ${{fromJson(needs.setup-matrix.outputs.matrix)}}
```

Comment:
IMO, `client-network` is a bit misleading, since it's actually a node configuration. I'd suggest renaming it into something like `config` or `node`

Reply:
Chnaged